apps/condo/domains/miniapp/middlewares/launchB2BAppMiddleware/index.js

```diff
@@ -0,0 +1,132 @@
+const cookieParser = require('cookie-parser')
+const express = require('express')
+const { get } = require('lodash')
+
+const { getByCondition } = require('@open-condo/keystone/schema')
+
+const LaunchB2BAppHelpers = require('./helpers')
+
+
+class LaunchB2BAppMiddleware {
+    async prepareMiddleware () {
+        const helpers = LaunchB2BAppHelpers.getInstance()
+        const { config, jwksResponse, signLaunchParams } = helpers
+
+        const app = express()
+        app.use(cookieParser())
+
+        app.get('/api/miniapp/launch/jwks', (req, res) => {
```

Comment:
why you cant use existing oidc urls?

Reply:
We can

Reply:
This is the same thing if the same keys are used

Reply:
changed to redirect